homeassistant/components/tellduslive.py

```diff
@@ -360,3 +359,18 @@ def _last_updated(self):
         """Return the last update of a device."""
         return str(datetime.fromtimestamp(self.device.lastUpdated)) \
             if self.device.lastUpdated else None
+
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID."""
+        return "tellduslive-{}".format(self._id)
```

Comment:
Is this `self._id` different ?

Reply:
Yes, it is just the device number (as an integer).